pkg/controller/amazoncloudintegration/amazoncloudintegration_controller.go

```diff
@@ -132,51 +138,70 @@ func (r *ReconcileAmazonCloudIntegration) Reconcile(ctx context.Context, request
 			r.status.OnCRNotFound()
 			return reconcile.Result{}, nil
 		}
-		r.SetDegraded("Error querying AmazonCloudIntegration", err, reqLogger)
+		status.SetDegraded(r.status, operatorv1.ResourceReadError, "Error querying AmazonCloudIntegration", err, reqLogger)
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
 	r.status.OnCRFound()
+	// SetMetaData in the TigeraStatus like observedGenerations.
+	if instance != nil {
```

Comment:
Done. Removed in all place except logstorage controller since there is a test case which injects nil logstorage hence i assumed there might be a use case for nil logstorage